src/model/RestAction.ts

```diff
@@ -220,6 +230,24 @@ class RestAction implements Action {
                   );
         };
 
+        const getClientCertificationConfiguration = (
+            clientCertificatePath,
+            clientKeyPath,
+        ) => {
+            try {
+                const cert = readFileSync(clientCertificatePath);
```

Comment:
`readFileSync`does not accept `undefined` or `null`, at least according to it's type definition. Have you checked what happens if `clientCertificatePath` is not defined? It would probably a good idea to check against `undefined` before calling `readFileSync`.

EDIT: ignore this, this is checked before the function is called. As mentioned above, please add type annotations for the parameters so it is clear that they are always `string`s and not `undefined` or `null`.